src/main/ui/app/component/AddOnSearch.jsx

```diff
@@ -56,32 +56,45 @@ export default class AddOnSearch extends Component {
     }
 
     render() {
-        return (
-                <div>
-                    Type:
-                    <select onChange={event => this.setType(event.target.value)}>
-                        <option value="">All Types</option>
-                        <option value="OMOD">Module (OMOD)</option>
-                        <option value="OWA">Open Web App (OWA)</option>
-                    </select>
 
-                    <br/>
+const title = (
+<span>Type: {this.state.type}</span>
+);
+        return (
+                <div className="row col-md-12 col-sm-12 col-xs-12 pushdown">
+                    <div>
+    <div className="input-group">
+    
+      <div className="input-group-btn">
+       
+ <DropdownButton title={title}  id="dropdown-size-small" onSelect={event => this.setType(event)}>
+        <MenuItem eventKey="">All Types</MenuItem>
+        <MenuItem eventKey="OMOD" >Module (OMOD)</MenuItem>
+        <MenuItem eventKey="OWA">Open Web App (OWA)</MenuItem>
+      </DropdownButton>
+      </div>
+  
+  
                     <DebounceInput
                             placeholder="Search..."
-                            className="search"
+                            className="form-control col-md-10 col-sm-10 col-xs-10"
                             minLength={1}
                             debounceTimeout={500}
                             onChange={event => this.setQuery(event.target.value)}/>
-                    <i className="fa fa-search fa-2x"></i>
```

Comment:
Oh, good point, I noticed this also. As it stands now, there's no value to having the icons, so I think we could just remove them.

In the future I'd like to let each addon specify its own icon (in the add-ons-to-index.json file).

So, yes, for now it's fine to remove the icons.